sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedPlainValuesReader.java

```diff
@@ -53,19 +53,47 @@ public void skip() {
     throw new UnsupportedOperationException();
   }
 
+  private void updateCurrentByte() {
```

Comment:
Thanks, I would keep `updateCurrentByte` for now. Because public methods are using `read***` prefix, I chose `updateCurrentByte` to distinguish this private method.

Reply:
Yes, makes sense. Let's keep it as is. Thanks.